lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/components/SocialWarningListItem.kt

```diff
@@ -0,0 +1,75 @@
+package com.ustadmobile.libuicompose.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Warning
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.ustadmobile.core.MR
+import dev.icerock.moko.resources.compose.stringResource
+
+@Composable
```

Comment:
This must as per previous comment use the Jetpack Compose ListItem composable. That is what will provide consistent styles.

e.g. 
```
ListItem(
   headlineContent = { Text(MR.strings.be_careful_interacting_online) }, 
   supportingContent = {
       Column {
           Text(stringResource(MR.strings.be_careful_not_to_share)
           Row {
               //buttons can go here
           }
        }
   }
)
```

Reply:
![Screenshot from 2024-11-26 11-57-39](https://github.com/user-attachments/assets/fd458250-d407-4257-8d84-c429e2ac2fd0)

The revised version (as of this writing) is still clearly not consistent with the coding style:

As per https://github.com/UstadMobile/UstadMobile/blob/primary/CODING-STYLE.md#screens

"Screens must use default margins and colors provided by the platform theme (Jetpack Compose or MUI) unless specifically noted otherwise"

That means the text style, text color, and tint should NOT be manually set unless it is explicitly noted to do so.

Reply:
Additionally (as per [ListItem doc](https://developer.android.com/reference/kotlin/androidx/compose/material3/package-summary#ListItem(kotlin.Function0,androidx.compose.ui.Modifier,kotlin.Function0,kotlin.Function0,kotlin.Function0,kotlin.Function0,androidx.compose.material3.ListItemColors,androidx.compose.ui.unit.Dp,androidx.compose.ui.unit.Dp))) the icon should go in the leadingContent, you should not create a row within leadingContent or supportingContent.